usr/sbin/registercloudguest

```diff
@@ -341,12 +341,16 @@ def setup_ltss_registration(registration_target, regcode):
     else:
         base_product = get_product_triplet(product)
```

Comment:
This should be moved to line 347 of this PR, inside the no product name block, if we are going to have a default name

IMO, we should have both regcode + product_name from the cmdline options, instead of a default product_name, the user knows which product they got the regcode for

if extensions to be activated with a regcode will have to match the base product version/arch maybe we dont need the triplet ? will we support different versions than the base product for the extensions ?

Reply:
> This should be moved...
I rebased the change here with master and that move has already been done

> IMO, we should have both regcode + product_name from the cmdline options, instead of a default product_name, the user knows which product they got the regcode for
Well this is not the scope of this PR. With this PR we give the user an opportunity to specify the product on the cmdline which did not exist before. Using the base product name extended by 'LTSS' as the default product name if none was specified was a decision we wrote into the implementation details document and is not touched nor questioned in this PR. I believe the default is good and having an opportunity to specify it on the commandline is still a question I'd like to discuss.

So is there a real use case for this option in our scope ?

After all the LTSS testing I think there is no use case for allowing alternative product names and I believe LTSS should follow the base product only.

Reply:
> So is there a real use case for this option in our scope ?

I think there is, right now we have LTSS but the moment we have one more non free extension available to be activated, we would need to specify the product somehow, having `-p` in the cli is the right option, as it is the pattern we have been using

Reply:
True, you assumed that, I did assume if more products are added to be activated with `registercloudguest -r regcode` same way as LTSS, I do not know if that would be a use case in the future (I assumed other products may or may not follow) and if so, how they would be activated

Reply:
Yes this is also a fair assumption. However, I suggest to wait adding this functionality when we really have this use case. I'm going to close this PR but keep the branch for later.

I believe it's better not to provide an option which could be easily used to try a registration against a bogus product name and I also believe customers would require documentation to pass along the correct value. For the later a real use case is needed so let's stick to what we have right now.

Thanks